src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -75,6 +75,13 @@ public void noDatastore() {
         Assert.assertEquals(null, actual);
     }
 
+    @Test
+    //If the entity doesn't "timestamp" property then null should be returned
+    public void noTimeStampProperty() {
+        Object actual = timing_properties_test.getTimestampProperty("user", datastore);
```

Comment:
Just realized but we should be accepting the specific object type and not just Object here as well.